driver/razermouse_driver.c

```diff
@@ -1418,6 +1436,29 @@ static ssize_t razer_attr_read_logo_led_effect(struct device *dev, struct device
     return sprintf(buf, "%d\n", response.arguments[2]);
 }
 
+/**
+ * Write device file "mode_wave" (for extended mouse matrix effects)
```

Comment:
Please fix the comment

Reply:
I've fixed the comment here along with other comments to reference scroll.